SpatialGDK/SpatialGDK.uplugin

```diff
@@ -48,7 +48,7 @@
     {
       "Name": "SpatialGDKTests",
       "Type": "Editor",
-      "LoadingPhase": "Default",
+      "LoadingPhase": "PreLoadingScreen",
```

Comment:
Why this change?

Reply:
Some plugins are loaded as PreDefault. I'm capturing the setting available when the test plugin is loaded through a gloabl variable(although I could do it in the module initialization function, to be sure), and I'm checking that its value is the expected one in the tests.
If the test pass that way, it should ensure that any plugin loaded from the PreDefault stage has access to the correct value of the flag.

Reply:
I wonder if this will have impacts on future tests we want to write. @aleximprobable do you have an opinion?

Reply:
No. However, I think it should be fine as long as existing tests pass. And if it breaks future tests - we can revisit this decision.